OpenMRS-iOS/AppDelegate.m

```diff
@@ -31,6 +32,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 // SignInViewController *vc = [[SignInViewController alloc] init];
 //    [[[KeychainItemWrapper alloc] initWithIdentifier:@"OpenMRS-iOS" accessGroup:nil] resetKeychainItem];
     [Instabug startWithToken:@"9827bbb908adffd0d628b2b1f6890899" captureSource:IBGCaptureSourceUIKit invocationEvent:IBGInvocationEventShake];
+    [Flurry startSession:@"ZTXS4QTSX5499GPKQ6G4"];
```

Comment:
This is your key, correct? We'd have to decide who we want having analytics to the app. @harshadura input?